mvp-spec/audit-log-design.md

```diff
@@ -135,14 +98,56 @@ the Contracting Party handles those objects.
+```
+</details>
 
-Once the user has clicked on the Audit Button, she/he is redirected to the
-Audit UI. The Contracting Party generates the UI based on the validations 
-of the Prebid SSO Data and the Transmission Results. The following elements must
+###  Alternative 2 - Javascript
+
+The Audit logs can be stored on the `window` of the webpage via `javascript`. The exact path is `window.paf.audit`. The `audit` object is a dictionary that provides the audit log (the value) given the Transaction Id (the key). 
+
+```javascript
+window.paf = {};
+window.paf.audit = {};
+window.pad.audit["a0651946-0f5b-482b-8cfc-eab3644d2743"] = audit_log1;
+window.pad.audit["eab36444-0f5b-482b-8cfc-eaba06519462"] = audit_log2;
+```
+
+
+## Display of the Audit UI
+
+## Interface
+
+Once the user has clicked on the Audit Button, the Audit UI appear as a popup. 
+The Contracting Party generates the UI based on the validations of the 
+Prebid SSO Data and the Transmission Results. The following elements must
 appear in the page:
 
 | Element                          | Details                                   |
```

Comment:
I don't find instructions on how to represent preferences and ids, apart from the green / red dot. Shouldn't we define **what** must be displayed for preferences and ids?